.eslintrc.json

```diff
@@ -22,16 +28,28 @@
       "error",
       // Allow functions to include the full signature of a protocol they adhere
       // to, even if they don’t use some args.
-      {"args": "none"}
+      {
+        "args": "none"
+      }
     ],
     "quotes": [
       "error",
       "single"
     ],
+    "object-curly-spacing": [
+      "error",
+      "never"
+    ],
```

Comment:
Do you have thoughts on whether `"never"` or `"always"` is the right value here? Obviously the guide says `"never"`, but we have *quite* a mix across the codebase and I’ve heard some feedback advocating `"always"` instead. (cc: @SYU15)

Reply:
I personally prefer the look of the "always" rule ever so slightly, but am completely open to either. Should we take a vote?

Reply:
No need for voting. Feel free to switch to `always`.

Reply:
Okay! Changed to "always", and updated CONTRIBUTING.md accordingly

Reply:
Just saw this for some reason. "always" is preferable for me too.